test/end-to-end/signin.test.js

```diff
@@ -486,13 +486,15 @@ describe("SuperTokens SignIn", function () {
                 `${TEST_CLIENT_BASE_URL}/redirect-here`
             );
 
-            // test that if we visit auth again, we end up in redirect-heree again
+            // test that if we visit auth again, we end up in redirect-heree again with query params kept
             await Promise.all([
-                page.goto(`${TEST_CLIENT_BASE_URL}/auth?rid=emailpassword&redirectToPath=%2Fredirect-heree`),
+                page.goto(
```

Comment:
Please add a similar ones that test the redirection:
- emailpassword sign-in 
- emailpassword sign-up.
- passwordless sign-in
- thirdparty sign-in

This test only cheks that the auth pages redirect to the appropriate path if we navigate to it while already logged in.